### PR TITLE
[MNT] Updating the only tutorial that doesn't run on CI (Lazy benchmark)

### LINKED ISSUE #650: Loading/organising data tutorial is broken on dev
On the latest master, the example https://braindecode.org/dev/auto_examples/datasets_io/benchmark_lazy_eager_loading.html doesn't seem to work.

e.g.:

```
    for ds in windows_ds.datasets:
        ds.windows.drop_bad()
        assert ds.windows.preload == preload
```

Raises error:
```
    ds.windows.drop_bad()
    ^^^^^^^^^^
AttributeError: 'EEGWindowsDataset' object has no attribute 'windows'
```

Commenting those lines out, the next error appears at:
```
    n_channels = len(dataset.datasets[0].windows.ch_names)
    n_times = len(dataset.datasets[0].windows.times)
```

```
    n_channels = len(dataset.datasets[0].windows.ch_names)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'EEGWindowsDataset' object has no attribute 'windows'
```

I presume for the latter, `windows` needs to be renamed to `raw`. Not sure about the former.

### PR DESCRIPTION
**Pull request by bruAristimunha**
Closes #650

### CODE REVIEW

**Comment by bruAristimunha** [comment]
Can you run the tutorial again @itsaphel? I don't have the temple university dataset in my notebook and my cluster is not connected at my home, only in the laboratory.

File: examples/datasets_io/benchmark_lazy_eager_loading.py [hunk 1]
```diff
@@ -335,10 +336,9 @@ def run_training(model, dataloader, loss, optimizer, n_epochs=1, cuda=False):
         num_workers=num_workers,
         worker_init_fn=None,
     )
-
     # Instantiate model and optimizer
-    n_channels = len(dataset.datasets[0].windows.ch_names)
-    n_times = len(dataset.datasets[0].windows.times)
+    n_channels = len(dataset[0][0].shape[0])
+    n_times = len(dataset[0][0].shape[1])
```
Comment (itsaphel) [inline examples/datasets_io/benchmark_lazy_eager_loading.py:341]: I think the `len` causes this: `TypeError("object of type 'int' has no len()")`

maybe:
```
n_channels = dataset[0][0].shape[0]
n_times = dataset[0][0].shape[1]
```

Comment (bruAristimunha) [inline examples/datasets_io/benchmark_lazy_eager_loading.py:341]:
```suggestion
    n_channels = dataset[0][0].shape[0]
    n_times = dataset[0][0].shape[1]
```

**Comment by bruAristimunha** [comment]
Thank you @itsaphel

**Pull request merged**